fabtests/common/shared.c

```diff
@@ -2060,25 +2060,52 @@ ssize_t ft_inject(struct fid_ep *ep, fi_addr_t fi_addr, size_t size)
 	return ret;
 }
 
+/**
+ * @brief for write operations, get remote addr offset
+ * @param[in] buf	local pointer within the tx_buf region.
+ * @return an offset to the corresponding region in the remote's rx_buf
+*/
+static size_t ft_write_offset(char *buf)
```

Comment:
Drop the comment -- it's just extra maintenance cost.  Declare buf as const char to indicate that it's input.  Rename to ft_remote_write_offset() to clarify what the function returns.

Reply:
done.